Blizzard.IO.RabbitMQ.Tests/Rpc/NetqRpcRabbitClientTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using Blizzard.IO.RabbitMQ.Rpc;
+using EasyNetQ;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class RequestStub
+    {
+        public int Value { get; set; }
+    }
+
+    class RespondStub
+    {
+        public int Value { get; set; }
+    }
+
+    [TestFixture]
+    public class NetqRpcRabbitClientTests
+    {
+        private Mock<INetqRabbitRpcConnection<Func<Type, object>>> _netqRpcRabbitConnectionMock;
+        private Mock<ILogger> _logger;
+        private Mock<IBus> _busMock;
+
+        private NetqRabbitRpcClient _rpcClient;
+
+        [SetUp]
+        public void Setup()
+        {
+            _logger = new Mock<ILogger>();
+            _netqRpcRabbitConnectionMock = new Mock<INetqRabbitRpcConnection<Func<Type, object>>>();
+            _busMock = new Mock<IBus>();
+            _netqRpcRabbitConnectionMock.Setup(connection => connection.Bus).Returns(_busMock.Object);
+
+            _rpcClient = new NetqRabbitRpcClient(_netqRpcRabbitConnectionMock.Object, _logger.Object);
+        }
+
+        [Test]
+        public void Request_OnValidRequestAndAbstractMessageTypeBus_ShouldReturnValidRespone()
```

Comment:
I think that instead of `valid`, `certain` will be more precise here. Since `respond` here is, and should, be the only respond to `request`

Reply:
I dont think certain is the correct word I think I could just say `Respone` without the valid or certian

Reply:
F